src/Symfony/Bundle/FrameworkBundle/Templating/GlobalVariables.php

```diff
@@ -33,21 +34,29 @@ public function __construct(ContainerInterface $container)
     }
 
     /**
-     * Returns the current user.
-     *
-     * @return mixed
+     * Returns the current token.
      *
-     * @see TokenInterface::getUser()
+     * @return TokenInterface|null
      */
-    public function getUser()
+    public function getToken()
     {
         if (!$this->container->has('security.token_storage')) {
             return;
         }
 
-        $tokenStorage = $this->container->get('security.token_storage');
+        return $this->container->get('security.token_storage')->getToken();
+    }
 
-        if (!$token = $tokenStorage->getToken()) {
+    /**
+     * Returns the current user.
+     *
+     * @return mixed
```

Comment:
No, it's just that we don't add such annotations anymore. Old code should be kept as is.